apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/TracesResourceTest.java

```diff
@@ -105,13 +106,16 @@ class TracesResourceTest {
     public static final String USER = UUID.randomUUID().toString();
     public static final String WORKSPACE_ID = UUID.randomUUID().toString();
     private static final String TEST_WORKSPACE = UUID.randomUUID().toString();
+    private static final String TRUNCATE_TRACES = "TRUNCATE traces;";
```

Comment:
We shouldn't need to truncate it, as it ties the test to the table's current state. We can create new workspaces, so truncating is not required.

Reply:
Well, my test and endpoint definitely depends on the table state and it should,  considering the purpose for this endpoint. Also in general behavior of the system could/should depend on DB state.
Also for tests it's a common practice, that resources are cleaned up before/after the test. So one test is not affected by the others.

Reply:
I mean, the test should be independent of data from other tests. If you need complete isolation, you can create a separate test with a new Isolated database container. Let me know if you need help. We can do pair programming if you like.

Reply:
Updated as discussed